frontend/packages/dev-console/src/components/import/ImportForm.tsx

```diff
@@ -93,6 +93,7 @@ const ImportForm: React.FC<ImportFormProps & StateProps> = ({
       },
     },
     resources: Resources.Kubernetes,
+    resourceTypesNotValid: contextualSource ? [Resources.KnativeService] : [],
```

Comment:
We need to add this to `DeployImage.tsx` also,  Knative service should be removed from there as well.

Reply:
Damn, I thought I did a good reverse search... I guess I missed something. Good catch; thanks @karthikjeeyar.